### PR TITLE
TCR-31: mod-lists

### PR DESCRIPTION
**Pull request by todolson**
Initial technical review of mod-lists.

### CODE REVIEW

File: module_evaluations/TCR-31_2023-09-06-mod-lists.MD [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+* [x] The module responds with a tenant's content based on x-okapi-tenant header
+* [x] Standard GET `/admin/health` endpoint returning a 200 response
+  * _Note: read more at https://wiki.folio.org/display/DD/Back+End+Module+Health+Check+Protocol_
+  * _Note: yes, via actuator_
+* [x] High Availability (HA) compliant
+  * Possible red flags:
+    * Connection affinity / sticky sessions / etc. are used
+    * Local container storage is used
+    * Services are stateful
+* [ ] Module only uses infrastructure / platform technologies on the [officially approved technologies](https://wiki.folio.org/display/TC/Officially+Supported+Technologies) list.
```
Comment (marcjohnson-kint) [inline module_evaluations/TCR-31_2023-09-06-mod-lists.MD:60]: Which technology is used that isn't considered part of the list?

Is it min.io?

Comment (todolson) [reply module_evaluations/TCR-31_2023-09-06-mod-lists.MD:60]: Right, Minio isn't actually on a list and [DR-000026 - S3/MinIO](https://wiki.folio.org/pages/viewpage.action?pageId=96419874) has been sitting in draft for a long time. @craigmcnally and I agree that while TC is probably fine with this, we really need to discuss make an explicit decision for this TCR, as will as prod the DR along.

Comment (marcjohnson-kint) [reply module_evaluations/TCR-31_2023-09-06-mod-lists.MD:60]: Practically, it is on the list because we've got modules already in use that use it

**Comment by todolson** [comment]
Have updated TCR-31 to include notes on the unchecked criteria.

**Comment by todolson** [comment]
## Recommendation and justification
Reviewers recommend acceptance with discussion of three points.
With the conditional acceptance of `mod-fqm-manager`, there are three criteria which are in question:
* [ ] Module descriptor MUST include interface requirements for all consumed APIs
  * `mod-lists` uses `/query` and `/entity-types` APIs, module descriptor needs to be updated
  * Reviewers think this is a trivial fix and in the context of our timeframe should not block acceptance.
  * (This was the source of some controversy in the TC discussion.)
* [ ] Sonarqube hasn't identified any security issues, major code smells or excessive (>3%) duplication
  * There are code smells related to Exception classes having 6 parents which is greater than 5 authorized. Reviewers would give it a pass, but would like TC to discuss.
  * Ran out of time to discuss, but did not generate controversy.
* [ ] Module only uses infrastructure / platform technologies on the [officially approved technologies](https://wiki.folio.org/display/TC/Officially+Supported+Technologies) list.
  * _Note: need to discuss with the TC whether or not MinIO should be on the the OST list_
  * Also generated no controversy, [DR-000026 - S3/MinIO](https://wiki.folio.org/pages/viewpage.action?pageId=96419874) has been around in DRAFT status for over two years and seems like an oversight.

**Review by craigmcnally — APPROVED** [review]

**Pull request merged**